docs/multicluster/quick-start.md

```diff
@@ -74,62 +86,177 @@ antrea-multicluster   deployment.apps/antrea-mc-controller   1/1     1
 kube-system           deployment.apps/antrea-mc-controller   1/1     1            1           48s
 ```
 
-### Step 2 - initialize ClusterSet
+#### Step 2 - initialize ClusterSet
+
+Run the following commands to create a ClusterSet with cluster A to be the
+leader, and also join the ClusterSet as a member.
+
+```bash
+antctl mc init --clusterset test-clusterset --clusterid test-cluster-leader --create-token -o join-config.yml
+antctl mc join --clusterid test-cluster-leader -n kube-system --config-file join-config.yml
```

Comment:
When the `--config-file` is provided, we need provide clusterID in the the file instead of `--clusterid`.

Reply:
In the generated config file, we set `clusterID` and `namespace` to "". Will unmarshal still reset the field? If that is the case, maybe we should not add `clusterID` and `namespace` to the config file, or even better add them and other optional fields as comments.
@hjiajing

Reply:
Yes. If the `clusterID` is empty in the config file and specify it in the command line. It will reset when unmarshal. Maybe we can add some comments above the output config file to remind the users to edit the config file or delete the fields with empty value?